docs/tutorial/index.html

```diff
@@ -12,7 +12,7 @@
 </script>
   <link href="webpages/css/main.css" rel="stylesheet" type="text/css">
   <link rel="icon" type="image/png" href=
-  "https://checkerframework.org/favicon-checkerframework.png">
+  "https://eisop.github.io/favicon-checkerframework.png">
```

Comment:
This URL doesn't seem to work.
I want to update to a different icon anyways, so let's ignore this. Can you file an issue so we don't forget?